torch/csrc/generic/methods/TensorMath.cwrap

```diff
@@ -757,6 +757,36 @@
       return: accreal
       arguments:
         - THTensor* self
+        - arg: bool unbiased
+          negated: True
+    - cname: varall
+      return: accreal
+      arguments:
+        - THTensor* self
+        - CONSTANT false
```

Comment:
I'm adding `default: true`, but it gets replaced with `True` during code generation. May I keep it as `default: 1`, which works?